generators/app/templates/boilerplate/gulp/gulp-config.js

```diff
@@ -1,13 +1,14 @@
+/*eslint quotes: ["error", "single"]*/
```

Comment:
We only have a handful of true source js files that are generated by the templates. This line just enforces that we have single quotes in our source JS files (since eslint is no longer checking quotes & TSLint doesn't look at js.

The line below is a pointer to an issue that's being tracked in the Typescript repo to enable users to specify a flag/config/etc. to typescript compiler so that we can control whether or not it emits single or double quotes on generated code